docs/js/features/odata/use-vdm.md

```diff
@@ -0,0 +1,277 @@
+---
```

Comment:
General remark: Try to make separate pull requests for separate concerns, that allows for different topics to be reviewed and merged independently. ;)

Reply:
Will do next time!